generate.go

```diff
@@ -9,9 +9,8 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/ocitools/libgen"
```

Comment:
On Mon, Jul 11, 2016 at 07:09:30AM -0700, hmeng-19 wrote:
> > @@ -9,9 +9,8 @@ import (
> >  	"strconv"
> >  	"strings"
> >  
> > -	"github.com/Sirupsen/logrus"
> > +	"github.com/opencontainers/ocitools/libgen"
>
> @wking , this new package structure looks very different from the
> current package structure…

Yeah, but if we're going to restructure things, I think we want to
take a big step to get to where we (think we) want to be, instead of
taking a bunch of short steps.  The big step may not be perfect, but
it's got a better chance at being right than a small step after which
we already plan on making further changes.